examples/persistence_service/c++11/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+# (c) 2022 Copyright, Real-Time Innovations, Inc. All rights reserved.
+# No duplications, whole or partial, manual or electronic, may be made
+# without express written permission.  Any such copies, or revisions thereof,
+# must display this notice unaltered.
+# This code contains trade secrets of Real-Time Innovations, Inc.
+
+cmake_minimum_required(VERSION 3.11)
+project(rtiexamples-persistence-service)
+list(APPEND CMAKE_MODULE_PATH
+    "${CMAKE_CURRENT_SOURCE_DIR}/../../../resources/cmake/Modules"
+)
+include(ConnextDdsConfigureCmakeUtils)
+connextdds_configure_cmake_utils()
+
+# Include ConnextDdsAddExample.cmake from resources/cmake
+include(ConnextDdsAddExample)
+
+connextdds_add_example(
+    IDL "hello_world"
+    LANG "C++11"
+)
+
+find_package(
```

Comment:
Hi @pralhadsapre, ensure the `find_package(RTIConnextDDS` call is before the `include(ConnextDdsAddExample)` line, so you use the same components for the `connextdds_add_example`.

Reply:
Good point. Done!